### PR TITLE
Add package 'Copy Files and Folders'

### PR DESCRIPTION
**Pull request by toastwaffle**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read the docs.
- [x] I have tagged a release with a semver version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [ ] My package doesn't add context menu entries.
- [x] My package doesn't add key bindings.

My package adds a "Copy..." entry to the Side Bar context menu for copying files and folders

There are no packages like it in Package Control.

### CODE REVIEW

File: repository/c.json [hunk 1]
```diff
@@ -3807,7 +3818,6 @@
 		{
 			"name": "CopyToSharePointRoot",
 			"details": "https://github.com/joladev/SublimeCopyToSharePointRoot",
-			"author": "eakron",
```
Comment (braver) [inline repository/c.json:3810]: This looks like an unintentional change?

**Comment by braver** [comment]
[SideBarTools](https://packagecontrol.io/packages/SideBarTools) already provides this functionality (via "duplicate"). It does a few additional things too of course, but I'm not sure we need separate packages for every thing it does. What do you think?

File: repository/c.json (hunk 1 above)
Comment (toastwaffle) [reply repository/c.json]: Whoops, fixed!

**Comment by toastwaffle** [comment]
> [SideBarTools](https://packagecontrol.io/packages/SideBarTools) already provides this functionality (via "duplicate"). It does a few additional things too of course, but I'm not sure we need separate packages for every thing it does. What do you think?

Aha! I was sure something must exist for this already. It doesn't come up in the install menu when you search for "copy", which I guess is why I didn't see it.

**Comment by braver** [comment]
>  It doesn't come up in the install menu when you search for "copy", which I guess is why I didn't see it.

That's a really good point actually. I think I need to put some more labels on that package or otherwise improve its "SEO".

**Pull request closed without merging**